mlflow/llama_index/tracer.py

```diff
@@ -44,17 +46,40 @@ def __init__(self, id_: str, parent_id: Optional[str], mlflow_span: LiveSpan):
         self._mlflow_span = mlflow_span
 
 
+def _end_span(client: mlflow.MlflowClient, span: LiveSpan, status=SpanStatusCode.OK, outputs=None):
+    """An utility function to end the span or trace."""
+    if isinstance(outputs, (StreamingResponse, AsyncStreamingResponse)):
```

Comment:
Why isn't this consistent with `is_streaming_result`?

Reply:
That said I realized `StreamingAgentChatResponse` should also be in the list. Thanks for raising the point.